docs/plugin_developers_guide/02_writing_plugins.rst

```diff
@@ -385,6 +389,52 @@ We have used the properties *type* and *default* to define properties of the con
 
 Of the above properties of a configuration item *type*, *default* and *description* are mandatory, all other may be omitted.
```

Comment:
We may add a note directive here!

And for text `all other may be omitted.` Better to say all other are optional